src/kvstore/kvstore_local.h

```diff
@@ -135,6 +135,13 @@ class KVStoreLocal : public KVStore {
     PullRowSparseImpl(keys, val_rowids, priority);
   }
 
+  void SetCompress(const std::string& compress, const float neg_threshold,
+                   const float pos_threshold) override {
+    compress_ = compress;
+    pos_threshold_ = pos_threshold;
```

Comment:
Should ther ebe a check that neg_threshold_ is, indeed, smaller than pos_threashold?

Reply:
I do check that neg_threshold is less than 0 and pos_threshold is greater than 0 in python (kvstore.py). But checking here would make it more general yes. I'll add checks

Reply:
I wasn't implying the check should be made here -- I just thought of it here.  If you're checking it somewhere that will catch it early, that's fine with me.